### ISSUE TITLE
`brackets` catches not only usage at selectors

### ISSUE THREAD

**Issue by kizu** [rossPatton/stylint#144]
In Stylint 1.0.9 `"brackets": "never"` gives warnings not only for usage with selectors, but also in some cases where the curly braces are mandatory.

1. Hashes that are written inline.

    ``` styl
$b = { "bar": "baz" }
```

3. Nested hashes.

    ``` styl
$foo = {
      bar: {
        baz: raz
      }
}
```

2. Complex cases of interpolation for selectors.

    ``` styl
{ 'foo' }
      width: 10px
```

    ``` styl
{foo() + 'bar'}
      width: 10px
```

    etc.

**Comment by rossPatton** [comment]
1.0.10 is out now, this issue should be fixed. Lemme know if you still have issues.

**Comment by SimenB** [comment]
I have the inverse problem. We have `always`, and this triggers on variable-declarations (we have a file with only colors).

**Comment by rossPatton** [comment]
latest version should address this
